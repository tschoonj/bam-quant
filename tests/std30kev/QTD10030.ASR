File: C:\xrfdaten\2014KW30\chris\std30kev\std10030.spe
Start: 22.07.2014 10:08:50
Stop: 22.07.2014 10:10:30
BESSY: 297.9605076 298.0979299
IONI: 1.983764E-11 1.963301E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97980
Run: 100000
Dead: 2
ICR: 610
OCR: 0
Accumulated BESSY: 59715.1438929
Accumulated Ioni: 3.95259011000001E-09
N Accumulation: 200
Normfactor:      0.51642919
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zn
exp_x 132.720397949219 132.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  30   1      8.631     25465.       160.       1.12
