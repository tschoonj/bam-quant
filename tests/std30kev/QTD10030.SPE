$DATA:
0 2000
23154
14782
6271
1801
380
54
8
0
1
0
0
0
0
0
0
0
3
14
16
28
23
22
19
20
17
18
19
11
20
12
17
18
12
12
13
13
20
16
9
20
12
25
12
11
6
11
14
11
15
15
11
19
9
14
9
11
7
7
10
12
12
11
12
12
11
8
8
7
5
9
8
9
20
11
18
10
8
12
9
7
6
12
13
3
11
9
4
12
12
13
12
10
5
15
6
8
6
7
5
7
11
15
10
12
9
5
9
10
9
6
6
8
13
7
5
12
12
11
5
8
8
8
12
11
11
7
10
10
6
7
12
7
8
8
7
11
3
8
6
5
8
11
10
7
5
4
4
11
8
8
9
5
7
8
7
15
12
9
11
10
3
8
10
5
8
4
8
10
7
9
3
3
5
8
6
7
4
6
10
12
8
10
7
8
8
6
8
6
7
5
5
9
8
7
10
3
8
7
11
2
5
5
4
1
4
2
3
5
3
6
4
3
10
5
7
6
3
5
5
1
4
6
6
2
2
5
1
4
0
5
5
6
1
4
7
3
3
6
3
5
3
5
4
7
2
1
6
2
4
2
3
4
3
3
7
3
5
3
7
4
2
1
2
3
7
5
2
5
8
9
3
2
7
9
6
4
3
2
6
3
2
6
3
5
4
1
6
3
4
5
8
2
3
7
10
4
6
4
2
6
1
5
4
1
2
5
1
4
4
5
4
6
4
9
4
12
10
8
10
7
10
11
4
5
6
4
3
7
6
3
2
5
5
3
2
3
6
4
11
8
9
16
11
12
8
7
9
7
3
4
4
5
4
4
5
3
3
5
3
4
3
4
5
6
7
4
8
5
7
6
7
8
7
8
2
5
4
3
4
6
6
5
5
9
8
12
4
9
11
6
12
12
4
7
6
13
8
5
5
9
7
13
11
8
7
6
12
13
15
8
12
13
15
18
22
27
36
57
108
172
341
555
904
1331
1773
2350
2699
2785
2877
2669
2215
1681
1166
807
491
249
145
74
28
25
13
14
13
13
22
12
17
11
12
7
16
15
16
17
17
21
27
32
24
39
34
50
62
61
94
146
215
393
531
825
1107
1516
1739
1987
2072
2028
1816
1376
1172
745
509
317
177
75
47
11
10
10
1
3
2
3
1
3
2
0
0
2
1
1
3
2
0
0
2
1
0
0
2
2
3
2
0
3
1
2
0
3
0
1
0
1
1
1
0
0
0
1
2
0
0
0
1
1
2
1
1
1
1
0
1
1
1
0
1
1
1
1
1
2
2
1
2
2
2
0
2
2
1
2
4
1
0
1
4
3
0
1
2
0
2
1
1
2
1
1
1
2
2
2
0
0
2
3
6
1
5
0
2
2
2
1
1
4
1
1
0
2
2
0
1
6
0
7
1
0
3
2
0
3
3
1
3
1
3
3
5
2
2
2
0
3
5
2
3
5
3
7
3
2
1
3
1
4
4
6
3
3
5
2
3
3
2
5
2
2
3
1
0
0
0
4
5
7
3
1
3
2
1
2
1
0
1
3
2
1
6
4
1
2
4
4
2
6
8
0
4
1
3
4
6
4
4
5
3
2
6
3
4
3
8
2
1
4
8
3
7
9
5
2
4
2
1
5
5
3
6
2
5
6
4
4
2
2
5
7
6
2
3
2
3
4
4
7
3
4
2
3
1
1
3
2
5
5
3
3
3
2
5
6
4
6
6
8
4
4
5
8
7
2
4
3
3
7
6
3
2
2
1
5
7
3
7
3
9
5
3
6
5
10
15
12
22
24
34
45
55
57
64
71
70
86
56
57
42
38
41
24
18
14
9
6
4
8
7
4
5
2
1
1
3
2
2
7
1
2
5
3
2
3
5
3
4
3
5
0
3
7
5
3
3
5
2
1
8
4
2
6
1
7
2
2
3
2
3
4
3
0
3
4
3
1
7
4
6
5
2
1
3
3
5
3
1
2
4
1
4
4
5
5
3
8
7
2
9
9
5
7
6
11
9
16
17
17
15
14
16
11
16
7
9
11
12
4
4
5
8
9
3
3
3
5
4
4
4
5
4
3
7
7
4
3
5
3
1
4
4
6
2
3
3
2
7
4
5
3
3
3
4
3
2
5
2
7
1
3
2
1
5
2
6
4
0
6
4
1
0
5
1
11
7
5
2
4
3
4
3
3
2
4
5
5
2
4
1
1
2
4
5
5
4
3
6
1
4
4
3
0
4
3
4
2
2
1
2
4
2
2
1
0
2
2
6
6
1
4
2
5
4
2
2
1
4
2
3
5
4
1
2
1
5
2
4
4
1
3
1
3
1
2
2
3
7
5
2
3
3
3
2
3
4
0
4
3
2
3
2
5
2
4
3
1
3
4
2
3
5
5
1
2
0
3
5
3
4
2
3
2
4
0
1
4
9
7
7
0
4
1
1
1
6
5
4
7
4
5
4
1
2
2
4
2
3
1
1
2
5
4
2
1
3
2
3
2
3
5
2
4
2
5
2
6
1
3
3
1
4
3
1
6
1
2
4
3
0
3
0
3
4
1
4
1
3
2
0
3
1
3
3
2
0
1
2
2
1
2
5
2
7
3
9
6
7
11
9
9
8
5
19
11
15
9
16
21
18
15
20
16
17
17
14
12
13
5
9
6
4
4
5
6
4
3
3
2
2
2
3
3
2
4
1
1
0
2
2
2
2
2
2
3
1
1
2
2
2
1
1
0
2
1
3
1
3
4
0
4
1
2
2
0
2
1
2
2
0
0
3
1
2
1
1
2
3
5
1
0
0
3
3
0
0
1
0
0
2
2
5
2
4
0
1
2
2
0
6
6
6
4
6
8
12
7
8
13
8
5
8
11
7
16
15
9
14
11
11
5
7
8
7
4
3
5
2
7
2
4
1
1
5
3
0
2
1
0
1
1
2
0
5
1
1
1
1
2
2
4
3
4
5
8
5
3
4
5
1
1
5
5
1
3
3
1
2
1
3
1
1
1
1
1
5
1
2
0
4
2
2
0
0
0
2
1
3
0
2
2
1
1
3
0
0
3
1
0
2
0
0
0
1
4
1
3
2
2
4
2
1
2
2
4
1
1
1
3
3
3
0
4
4
1
2
1
3
2
1
3
5
3
1
3
2
0
4
5
3
2
1
0
2
1
0
3
1
3
2
1
0
2
3
3
1
1
1
3
3
0
2
2
5
2
2
4
4
2
5
5
0
2
5
3
5
5
4
6
6
2
3
1
2
5
2
3
4
0
3
4
2
1
0
4
3
4
4
5
2
1
1
3
6
2
2
5
2
1
3
6
3
4
5
4
3
6
4
3
3
3
2
4
7
2
0
5
3
3
4
3
2
3
4
3
4
4
4
3
5
1
5
8
7
6
6
3
2
2
3
4
4
4
7
2
3
2
4
3
3
5
5
2
6
4
5
2
5
6
1
6
0
7
5
4
3
6
2
7
4
9
7
6
4
3
2
6
4
10
4
11
4
1
6
7
6
6
7
10
9
9
7
11
6
8
9
8
8
7
5
5
13
3
5
9
9
11
10
11
6
16
5
11
8
7
13
9
11
8
7
14
10
12
20
10
18
17
13
14
12
17
13
21
9
18
28
12
27
12
17
16
18
17
19
18
22
12
11
20
26
17
17
14
14
21
25
15
17
25
27
21
22
25
24
21
28
23
29
29
27
23
20
26
25
23
18
27
25
17
26
17
30
26
25
25
29
27
16
22
26
33
23
30
26
25
30
26
16
27
22
27
26
18
23
22
29
25
24
20
24
30
27
13
26
23
19
28
27
18
28
20
18
27
20
27
22
15
18
14
23
18
23
26
19
18
11
19
19
23
22
16
12
18
19
20
24
18
19
12
16
13
16
13
25
20
12
20
15
19
9
28
21
24
15
13
16
15
19
18
20
15
26
20
29
22
25
20
35
33
20
38
46
43
47
57
37
54
58
57
48
56
54
54
54
57
68
47
49
51
40
44
45
43
55
46
46
41
29
31
30
26
25
27
14
20
22
15
15
18
9
13
8
6
3
5
4
4
2
3
0
0
0
0
2
0
0
2
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10030.spe
Start: 22.07.2014 10:08:50
Stop: 22.07.2014 10:10:30
BESSY: 297.9605076 298.0979299
IONI: 1.983764E-11 1.963301E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97980
Run: 100000
Dead: 2
ICR: 610
OCR: 0
Accumulated BESSY: 59715.1438929
Accumulated Ioni: 3.95259011000001E-09
N Accumulation: 200
Normfactor:      0.51642919
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zn
 
exp_x 132.720397949219 132.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  30   1      8.631     25465.       160.       1.12
