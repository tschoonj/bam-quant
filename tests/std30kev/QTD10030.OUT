     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:48  บ   
     บ  Spectrum: QTD10030.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        3 iterations done    บ   
     บ  ChiSquare =     5.1   last change =   .01%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -36.2 ๑   100.0   บ     -36.2 ๑     6.5   บ   
     บ  GAIN (eV/ch)  บ     20.235 ๑   2.000   บ    20.235 ๑    .014   บ   
     บ det NOISE (eV) บ       41.1 ๑    40.0   บ      41.1 ๑    56.8   บ   
     บ  FANO factor   บ       .142 ๑    .050   บ      .142 ๑    .017   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Zn-Ka                             25465. ๑    160.           บ   
     บ    KA1         8.639    .66048       16819. ๑    106.      1.1  บ   
     บ              428.730    167.07          41.           9.65E-01  บ   
     บ    KA2         8.616    .33952        8646. ๑     55.      1.3  บ   
     บ              427.593    166.86          41.           9.65E-01  บ   
     บ                                                                 บ   
     บ  2 Zn-Kb                             18865. ๑    138.           บ   
     บ    KB1         9.572   1.00000       18865. ๑    138.      2.9  บ   
     บ              474.839    175.34          37.           9.74E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                              39. ๑     10.           บ   
     บ    L1N2       18.203    .59701          23. ๑      9.       .9  บ   
     บ              901.395    238.63          38.           9.77E-01  บ   
     บ    L1O3       17.262    .40299          16. ๑      7.      1.6  บ   
     บ              854.900    232.57          33.           9.84E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
